Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -504,14 +504,16 @@ - (void)updateBarButtonItems
         UIImage *avatarImage = (localProfileAvatarImage
                 ?: [[[OWSContactAvatarBuilder alloc] initForLocalUserWithDiameter:kAvatarSize] buildDefaultImage]);
         OWSAssertDebug(avatarImage);
-        AvatarImageView *avatarView = [[AvatarImageView alloc] initWithImage:avatarImage];
```

Comment:
I'm not entirely sure why, but when voiceover is activated, you cannot select a custom non-button view, even though it's "tappable". 

Using a UIButton instead works.